### ISSUE TITLE
.template-lintrc is only loaded from the root of the first project directory in the project

### ISSUE THREAD

**Issue by emoryy** [binhums/linter-ember-template#12]
I think the best would be, if for any template file the closest .template-lintrc file would be loaded.
I.e. first check the directory of the template file, then traverse the directory tree upwards to the root, until a .template-lintrc.js is found.

**Comment by binhums** [comment]
Agreed. I seem to remember some logic of this description existing in ember-template-lint itself.

@emoryy would you be interested in jamming on an implementation with me?

**Comment by emoryy** [comment]
@binhums I couldn't find anything regarding this in `ember-template-lint`
But I whipped up my solution [here](https://github.com/emoryy/linter-ember-template/commit/7570a6c89788c2e0491b6387652fc7b85c9fbe53)
It seems to work, but I don't know how much more should I check for possible errors.
Also maybe the found file path could be cached somehow, so that we don't search for it with every modification.

**Comment by binhums** [comment]
Thanks for submitting the PR, I'm pleased with the way this is going.

Could you clarify your use case here? I'm assuming you have a project open in atom which contains different template lint config files in different directories? Are these directories individual ember projects?

The ember-cli-template-lint project assumes that there is a template lint config in the root of a given ember project:

https://github.com/rwjblue/ember-cli-template-lint/blob/master/index.js#L36

**Comment by amk221** [comment]
This situation affects me too.
`main-app/ember`
The project is `main-app/`, but the ember app (and its related template-lintrc.js file) is in `ember/`
